merlin/systems/dag/ops/workflow.py

```diff
@@ -13,19 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import json
-import pathlib
 from typing import List
 
-from merlin.core.protocols import Transformable  # noqa
 from merlin.dag import ColumnSelector
-from merlin.schema import ColumnSchema, Schema
-from merlin.systems.dag.ops.operator import PipelineableInferenceOperator  # noqa
-from merlin.systems.triton.conversions import (
-    dict_array_to_triton_request,
-    triton_response_to_dict_array,
-)
-from merlin.systems.triton.export import generate_nvtabular_model
```

Comment:
Love to see this clean-up happening. You can tell how much decoupling is happening just by reading the imports. ❤️